R/generic.R

```diff
@@ -133,13 +133,37 @@ check_generic <- function(fun) {
     stop("`fun` must contain a call to `R7_dispatch()`", call. = FALSE)
   }
 }
+
+#' Find a call in expression
+#'
+#' Find a call (namespaced or plain) in an language object.
+#'
+#' @param x An language object in which to search for a particular function.
+#' @param name A quoted function name to search for. Both the plain function
+#'   name and qualified function name from the package namespace are considered.
+#' @return The `call` object if found, or `NULL` otherwise.
+#' @keywords internal
+#'
 find_call <- function(x, name) {
```

Comment:
I'd suggest copying the interface to `rlang::is_call` here, and adding an `ns` argument that's `NULL` by default